### PR TITLE
Arbtitrum FE

### PR DESCRIPTION
**Pull request by Pljopi**

### CODE REVIEW

File: src/components/NetworkSelector/NetworkSelector.js [hunk 1]
```diff
@@ -25,6 +25,10 @@ function NetworkSelector(props) {
         value: "optimism",
         text: "optimism",
       },
+      {
```
Comment (imsaiph) [inline src/components/NetworkSelector/NetworkSelector.js:28]: I'd keep it out of the network selector for now until we're fully ready with it. Until then, we can manually type the url in.

File: src/components/NetworkSelector/NetworkSelector.js [hunk 2]
```diff
@@ -75,7 +80,7 @@ function NetworkSelector(props) {
           {options.v3.map((network) => (
             <DropdownItem
               key={`v3-${network.value}`}
-              active={currentVersion === "v3" && currentNetwork.value === network.value}
+              //        active={currentVersion === "v3" && currentNetwork.value === network.value}
```
Comment (0xCommanderKeen) [inline src/components/NetworkSelector/NetworkSelector.js:83]: why is this changed?

File: src/components/NetworkSelector/NetworkSelector.js [hunk 3]
```diff
@@ -48,10 +52,11 @@ function NetworkSelector(props) {
   return (
     <div>
       <Dropdown isOpen={dropdownOpen} toggle={toggle}>
-        <DropdownToggle caret tag="span" className={styles.dropdownToggle}>
-          <CryptoIcon size="1.5rem" className="me-2" name={currentNetwork.text} />
-          {currentVersion} {currentNetwork.text}
-        </DropdownToggle>
+        <DropdownToggle
```
Comment (0xCommanderKeen) [inline src/components/NetworkSelector/NetworkSelector.js:55]: why is this changed?

File: src/components/NetworkSelector/NetworkSelector.js [hunk 4]
```diff
@@ -48,10 +52,11 @@ function NetworkSelector(props) {
   return (
     <div>
       <Dropdown isOpen={dropdownOpen} toggle={toggle}>
-        <DropdownToggle caret tag="span" className={styles.dropdownToggle}>
```
Comment (0xCommanderKeen) [inline src/components/NetworkSelector/NetworkSelector.js:51]: why is this changed?

Comment (Pljopi) [reply src/components/NetworkSelector/NetworkSelector.js:51]: If you type in url  /v3/arbitrum and it calls currentNetwork it crashes if arbitrum is not in the network selector

File: src/components/NetworkSelector/NetworkSelector.js (hunk 2 above)
Comment (Pljopi) [reply src/components/NetworkSelector/NetworkSelector.js:83]: same, as above it crashes if arbitrum is not in network selector

Comment (Twigmaester) [reply src/components/NetworkSelector/NetworkSelector.js:83]: You can try to validate with a shorthand if statement, maybe something like this;

`{(currentVersion === "v3" && currentNetwork.value === network.value) ? network.value != arbitrum : null}`

**Review by tsifrer — APPROVED** [review]
